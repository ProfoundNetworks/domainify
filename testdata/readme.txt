www.example.com
www.profound.net
bogus
en.wikipedia.org
hosted.l.google.com
com
31.pool85-61-184.dynamic.orange.es
