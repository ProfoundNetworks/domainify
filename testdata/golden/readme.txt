example.com
profound.net

wikipedia.org
google.com

orange.es
